### PR TITLE
963 fix the previews

### PR DESCRIPTION
**Pull request by defong**
### Context
it bork

### Changes proposed in this pull request
Remove code rot, enforce rot code

### Guidance to review

Visit and all the links
https://rtt-review-pr-465.herokuapp.com/view_components

### CODE REVIEW

**Comment by defong** [comment]
[Fix the previews](https://trello.com/c/lcJbCMlK/963-fix-the-previews)

File: spec/features/view_components_spec.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    scenario "navigate to #{link}", skip: skip do
+      visit link
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_css("#main-content")
+    end
+  end
+
+  all_links.each do |link|
+    include_examples "navigate to", link, false
```
Comment (AbigailMcP) [inline spec/features/view_components_spec.rb:22]: Does this need to be `false` / `true` depending on whether it's being run locally or in CI?

**Review by AbigailMcP — COMMENTED** [review]
Yay working previews! They're beautiful.

Are the view component tests supposed to all run when you `rspec` locally? Is this what the `skip` option is for?

File: spec/features/view_components_spec.rb (hunk 1 above)
Comment (defong) [reply spec/features/view_components_spec.rb]: this should run regardless of CI, removed the skip options

Comment (AbigailMcP) [reply spec/features/view_components_spec.rb]: Cool! That makes sense 👌

**Review by AbigailMcP — APPROVED** [review]

**Pull request merged**